tests/fixtures/validate-form/form.ttl

```diff
@@ -0,0 +1,131 @@
+# Copied from an ember-submission-form-fields example:
```

Comment:
This doesn't include nested listings, so it doesn't fully cover the use-case, but didn't want to spend more time since I have other urgent tasks.

Reply:
LPDC and the new subsidy both contain nested listings, so we do encounter it there, but I tested it and those also validate properly. I just want a simplified example form that includes multiple listings but I can add that in the future (and I'll probably add it to submission-form-fields first).